launch/dense_stereo.launch

```diff
@@ -1,20 +1,20 @@
 <launch>
 
-<arg name="mav_name" default="ibis" />
+<arg name="mav_name" default="bluebird" />
 <arg name="namespace" default="$(arg mav_name)" />
-<arg name="first_camera_name" default="cam00" />
-<arg name="second_camera_name" default="cam01" />
+<arg name="first_camera_name" default="cam0" />
+<arg name="second_camera_name" default="cam1" />
 <arg name="scale" default="1" />
 <arg name="process_every_nth_frame" default="1" />
-<arg name="stereo_params_camchain" default="$(find mav_startup)/parameters/mavs/ibis/camchain_p22031.yaml"/>
-
+<arg name="stereo_params_camchain" default="$(find mav_startup)/parameters/mavs/bluebird/camchain_p21014.yaml"/>
 
 
+<group ns="$(arg namespace)">
   <node name="dense_stereo" pkg="image_undistort" type="dense_stereo_node">
-    <param name="input_camera_info_from_ros_params" value = "false"/>
+    <param name="input_camera_info_from_ros_params" value = "true"/>
```

Comment:
yeah I need to rewrite and clean up the example launch files and some of the docs